hail/python/hailtop/config/user_config.py

```diff
@@ -69,6 +68,17 @@ def configuration_of(section: str,
     return fallback
 
 
+def configuration_of(config_variable: ConfigVariable,
+                     explicit_argument: Optional[T],
+                     fallback: T,
+                     *,
+                     deprecated_envvar: Optional[str] = None) -> Union[str, T]:
+    config_variable_info = config_variables()[config_variable]
+    section = config_variable_info.section
+    option = config_variable_info.option
```

Comment:
> If we went with your suggestion, we'd have to have something like this. Which do you prefer?
> 
> ```python
> if '/' in config_variable.value:
>     section, option = config_variable.value.split('/')
> else:
>     section = 'global'
>     option = config_variable.value
> ```


I would prefer this. If I'm correct it would allow us to delete the repeated `section` and `option` information in `config_variabels`